src/renderer/coremods/commands/commands.ts

```diff
@@ -0,0 +1,424 @@
+              .map((plugin) => ({
+                name: plugin.manifest.name,
+                displayName: `${Messages.REPLUGGED_PLUGIN}: ${plugin.manifest.name}`,
+                value: plugin.manifest.id,
+              }))
+              .sort((a, b) => a.name.localeCompare(b.name)),
+          );
+          choices.push(
+            ...enabledThemes
+              .map((theme) => ({
+                name: theme.manifest.name,
+                displayName: `${Messages.REPLUGGED_THEME}: ${theme.manifest.name}`,
+                value: theme.manifest.id,
+              }))
+              .sort((a, b) => a.name.localeCompare(b.name)),
+          );
+
+          return choices;
+        },
+      },
+    ],
+    executor: async (interaction) => {
+      try {
+        const addonId = interaction.getValue("addon");
+        if (plugins.plugins.has(addonId)) {
+          await plugins.reload(addonId);
+        } else {
+          themes.reload(addonId);
+        }
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0x1bbb1b,
+              title: "Success", // TODO: i18n
+              description: Messages.REPLUGGED_COMMAND_RELOAD_MESSAGE_ENABLED.format({
+                type: plugins.plugins.get(addonId)
+                  ? Messages.REPLUGGED_PLUGIN
+                  : Messages.REPLUGGED_THEME,
+                name:
+                  plugins.plugins.get(addonId)?.manifest?.name ??
+                  themes.themes.get(addonId)?.manifest?.name,
+              }),
+            },
+          ],
+        };
+      } catch (err) {
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0xdd2d2d,
+              title: Messages.REPLUGGED_COMMAND_ERROR_GENERIC,
+              description: err as string,
+            },
+          ],
+        };
+      }
+    },
+  });
+  injector.utils.registerSlashCommand({
+    name: Messages.REPLUGGED_COMMAND_LIST_NAME,
+    description: Messages.REPLUGGED_COMMAND_LIST_DESC,
+    options: [
+      {
+        name: "send",
+        displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_SEND_NAME,
+        description: Messages.REPLUGGED_COMMAND_LIST_OPTION_SEND_DESC,
+        type: ApplicationCommandOptionType.Boolean,
+        required: false,
+      },
+      {
+        name: "type",
+        displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_TYPE_NAME,
+        description: Messages.REPLUGGED_COMMAND_LIST_OPTION_TYPE_DESC,
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: Messages.REPLUGGED_THEME,
+            displayName: `List themes`, // TODO: i18n
+            value: "theme",
+          },
+          {
+            name: Messages.REPLUGGED_PLUGIN,
+            displayName: `List plugins`, // TODO: i18n
+            value: "plugin",
+          },
+        ],
```

Comment:
Why do these have both name and displayName? What's the difference?

Also i18n

Reply:
name is used internally in code and displayName is shown. both required. well idk why i am doing different names and displayName. maybe there is one word limit on name. i dont really remember